velox/common/base/RuntimeMetrics.h

```diff
@@ -96,18 +93,13 @@ void addThreadLocalRuntimeStat(
 /// Scope guard to conveniently set and revert back the current stat writer.
 class RuntimeStatWriterScopeGuard {
  public:
-  explicit RuntimeStatWriterScopeGuard(
```

Comment:
The current code is good and we shouldn't assume the current writer runtime stats is always null. We might call into the other operator's method?